Day0/bipul018/assignment1/cjpt.c

```diff
@@ -0,0 +1,134 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+struct color{
+    unsigned char r;
+    unsigned char g;
+    unsigned char b;
+};
+int main(int argc,char *argv[])
+{
+
+    //default filenames will be used if not provided in command line
+    char source[100];
```

Comment:
You do not need to create a separate char array. You can directly use char * as follows 
```c
char * source = NULL;
if (argc > 1)
   source = argv[1];
else
   source = "render.txt";
```